### ISSUE TITLE
Improvements to support contact form

### ISSUE THREAD

**Issue by simison** [Trustroots/trustroots#2430]
**Goal:** improve the clarity of the support form and speed up the support team's process.

Note that any changes that pull information about people (avatar, name) cannot be done for the "logged out" version of the form for privacy reasons.

**Code**: 
- [Support module](https://github.com/Trustroots/trustroots/tree/5711b1da0b346a90f5f691e5d9c831ff40c36e6d/modules/support)
- [API for blocking users](https://github.com/Trustroots/trustroots/blob/0d00a492636eb06e309a38c1df3afb00f95c913d/modules/users/client/api/block.api.js)

## Visuals

The form is visible at https://www.trustroots.org/support?report=dummy

### Currently

**Form**

![image](https://user-images.githubusercontent.com/87168/137460878-449bfaf9-825d-4026-88c5-b99ec3bf4f93.png)

**Confirmation**

![image](https://user-images.githubusercontent.com/87168/137463286-41e10d39-e388-4d90-accf-ab5a1e221682.png)

**Logged out view**

![image](https://user-images.githubusercontent.com/87168/137479685-e9e60605-841e-4099-9f7d-cd01c56895ee.png)

### After

![image](https://user-images.githubusercontent.com/87168/137468533-290f6f21-bd50-44e9-850b-660e06b85c1f.png)

## Priority improvements

- [ ] Add checkbox to grant access to look into user's messages between them and whom they reported, checked by default:
  > "Allow support to see messages between the reported member and me." (Feel free to adjust copy)

    ![image](https://user-images.githubusercontent.com/87168/137466415-fffcbf13-500e-4366-be98-b99498213c87.png)

- [ ] Under _"See also"_ add _"Trustroots & safety"_ link once the page is done at `/safety`

- [ ] Under _"See also"_ add _"Blocking another member"_ link to `/faq` once the question is added in the FAQ.

## "Nice to have" improvements

Something that would make the experience better but isn't as important as the above things.

- [ ] Add ~10px spacing between the "see also" list

   ![image](https://user-images.githubusercontent.com/87168/137468244-9fdb7c5e-1ec6-434a-9928-5f8ecafa27a1.png)

- [ ] At the confirmation page, add "via email" to the copy for regular messages to:

    > "I’m just a small website robot but I’ve sent your message to our support people. Expect them to get back to you via email very soon!"

- [ ] At the confirmation page, change the copy to this or something along these lines when someone was reported, and remove the robot jokes. The point is to be empathic and indicates that we're really here to help and listen and align with the potential seriousness of the report.

    >  "Thank you for reporting this member to us. We're sorry to hear you didn't have a good experience with Trustroots. We will get back to you as soon as possible.". 

- [ ] Make the "report to police" badge more visible and move it up, right before `<form>` tag:

    ![image](https://user-images.githubusercontent.com/87168/137464589-78660ac2-e645-42fa-93be-eae6913d96be.png)

    ```html
    <p role="alert" class="alert alert-danger"><em>If you or someone you know have witnessed or been a victim of a crime, please report it to the police immediately.</em></p>
    ```

- [ ] Add "block this member from seeing and contacting me" checkbox, unchecked by default. See [form HTML docs](https://getbootstrap.com/docs/3.3/css/#forms)

    ![image](https://user-images.githubusercontent.com/87168/137465263-5405898f-ecc9-4448-af53-1d2b0036887a.png)

- [ ] Add name in addition to username. Assures a bit they're reporting the right person since they might not be familiar with the username, as it's less visible at the site.

    ![image](https://user-images.githubusercontent.com/87168/137463877-dc85a775-17f5-46fd-9cb7-21fd2e86369c.png)

- [ ] Add a profile picture

    ![image](https://user-images.githubusercontent.com/87168/137462834-6b9ecf09-ebb9-482c-b576-0de7d64abf14.png)

- [ ] Change "name + username"  into "You":

    ![image](https://user-images.githubusercontent.com/87168/137467638-4dd3dc97-6d20-4a97-908f-b24f358d3c99.png)

- [ ] Add "Incorrect email? [Change it](https://www.trustroots.org/profile/edit/account) so that we can contact you." text under email. You can use `<span class="help-block"></span>` HTML.

    ![image](https://user-images.githubusercontent.com/87168/137465760-d042cefd-77cd-433a-aaa5-accd7dfa251b.png)

- [ ] Add "I want to report someone" checkbox on a regular form when people just open support without reporting someone specifically.

    ![image](https://user-images.githubusercontent.com/87168/137467515-f034d175-7730-44c3-ab03-f214b9f10d97.png)

**Comment by simison** [comment]
@Plvtinum-1 I'm guessing that's Windows not liking `bin/script-name.js` instead of `node ./bin/script-name.js`. Changed it in https://github.com/Trustroots/trustroots/commit/9015b7d5095995338c349fe969b4863c8787d6b9, could you pull latest master and try again? Thanks for patience!

**Comment by simison** [comment]
@Plvtinum-1 I would recommend installing [Prettier](https://prettier.io/docs/en/editors.html) on your editor, and setting it to "format on save" (or "while typing"). 

If you prefer formatting manually, you can always run things manually with command `npm run reformat-files ./server.js`, which basically runs `prettier --write` for you.

Files also get formatted when you commit them on git.

**Comment by TMC89** [comment]
@noahsmindfuck @LudoB8 @Oded-lu @loberto: what do you guys think? :)

**Comment by loberto** [comment]
in my humble opinion, this would be a **freaking awesome** improvement!


If this is hard or complex to do all together, I would ask the checkbox for accessing their messages to be up and running ASAP.

**Comment by oneamanish** [comment]
see my pull req.

**Comment by loberto** [comment]
ping... anyone besides simison is on this issue?

**Comment by TMC89** [comment]
It might also be helpful to change this text: 
![Screenshot 2022-03-04 at 15 53 43](https://user-images.githubusercontent.com/74301087/156785816-ef62eda9-402f-41d8-bbc5-2fd8cde5ae66.png)
"...... Expect them to get back to you as soon as possible!"
